unittests/test_utility.py

```diff
@@ -765,111 +763,111 @@ def test_update(self):
             'a:b:c': {'k2': 5, 'k3': 6},
             '*': {'k1': 7, 'k3': 9, 'k4': 10}
         })
-        self.assertEqual(scoped_dict, scoped_dict_alt)
+        assert scoped_dict == scoped_dict_alt
 
 
-class TestReadOnlyViews(unittest.TestCase):
+class TestReadOnlyViews:
     def test_sequence(self):
         l = util.SequenceView([1, 2, 2])
-        self.assertEqual(1, l[0])
-        self.assertEqual(3, len(l))
-        self.assertIn(2, l)
-        self.assertEqual(l, [1, 2, 2])
-        self.assertEqual(l, util.SequenceView([1, 2, 2]))
-        self.assertEqual(list(reversed(l)), [2, 2, 1])
-        self.assertEqual(1, l.index(2))
-        self.assertEqual(2, l.count(2))
-        self.assertEqual(str(l), str([1, 2, 2]))
+        assert 1 == l[0]
+        assert 3 == len(l)
+        assert 2 in l
+        assert l == [1, 2, 2]
+        assert l == util.SequenceView([1, 2, 2])
+        assert list(reversed(l)) == [2, 2, 1]
+        assert 1 == l.index(2)
+        assert 2 == l.count(2)
+        assert str(l) == str([1, 2, 2])
 
         # Assert immutability
         m = l + [3, 4]
-        self.assertEqual([1, 2, 2, 3, 4], m)
-        self.assertIsInstance(m, util.SequenceView)
+        assert [1, 2, 2, 3, 4] == m
+        assert isinstance(m, util.SequenceView)
 
         m = l
         l += [3, 4]
-        self.assertIsNot(m, l)
-        self.assertEqual([1, 2, 2], m)
-        self.assertEqual([1, 2, 2, 3, 4], l)
-        self.assertIsInstance(l, util.SequenceView)
+        assert m is not l
+        assert [1, 2, 2] == m
+        assert [1, 2, 2, 3, 4] == l
+        assert isinstance(l, util.SequenceView)
 
-        with self.assertRaises(TypeError):
+        with pytest.raises(TypeError):
             l[1] = 3
 
-        with self.assertRaises(TypeError):
+        with pytest.raises(TypeError):
             l[1:2] = [3]
 
-        with self.assertRaises(TypeError):
+        with pytest.raises(TypeError):
             l *= 3
 
-        with self.assertRaises(TypeError):
+        with pytest.raises(TypeError):
             del l[:1]
 
-        with self.assertRaises(AttributeError):
+        with pytest.raises(AttributeError):
             l.append(3)
 
-        with self.assertRaises(AttributeError):
+        with pytest.raises(AttributeError):
             l.clear()
 
-        with self.assertRaises(AttributeError):
+        with pytest.raises(AttributeError):
             s = l.copy()
 
-        with self.assertRaises(AttributeError):
+        with pytest.raises(AttributeError):
             l.extend([3, 4])
 
-        with self.assertRaises(AttributeError):
+        with pytest.raises(AttributeError):
             l.insert(1, 4)
 
-        with self.assertRaises(AttributeError):
+        with pytest.raises(AttributeError):
```

Comment:
Can you introduce these changes separately with you upcoming PR about moving to pytest syntax? It's too hard to follow and off topic.

Reply:
oops, I retrieved the wrong file.